cluster-autoscaler/main.go

```diff
@@ -468,7 +468,7 @@ func registerSignalHandlers(autoscaler core.Autoscaler) {
 	}()
 }
 
-func buildAutoscaler(debuggingSnapshotter debuggingsnapshot.DebuggingSnapshotter) (core.Autoscaler, error) {
+func buildAutoscaler(debuggingSnapshotter debuggingsnapshot.DebuggingSnapshotter, context ctx.Context) (core.Autoscaler, *loop.LoopTrigger, error) {
```

Comment:
Context is usually passed as the first argument, can you reorder?

Reply:
Updated.